src/mumble/Themes.cpp

```diff
@@ -55,25 +63,59 @@ void Themes::applyFallback() {
 }
 
 bool Themes::applyConfigured() {
+	static QString currentThemePath;
+
+
 	boost::optional< ThemeInfo::StyleInfo > style = Themes::getConfiguredStyle(Global::get().s);
 	if (!style) {
 		return false;
 	}
 
-	const QFileInfo qssFile(style->getPlatformQss());
+	QString themePath;
+	if (style->themeName == "Auto" || style->name == "Auto") {
+#if QT_VERSION >= QT_VERSION_CHECK(6, 5, 0)
+		auto colorScheme = QGuiApplication::styleHints()->colorScheme();
+		if (colorScheme == Qt::ColorScheme::Dark) {
+			themePath = ":/themes/Default/Dark.qss";
+		} else {
+			themePath = ":/themes/Default/Lite.qss";
+		}
+#else
+		bool isDarkTheme = detectSystemTheme();
+		if (isDarkTheme) {
+			themePath = ":/themes/Default/Dark.qss";
+		} else {
+			themePath = ":/themes/Default/Lite.qss";
+		}
+#endif
+	} else {
+		if (style->name == "Dark") {
```

Comment:
This else is problematic, as this prevents the use of custom user-installed themes.